### PR TITLE
Styles mapcomponent and adds custom pins to mapcomponent

### PR DESCRIPTION
**Pull request by kierangillen**
- Fixes bug where map would sometimes not display
- Styles map component
- Adds custom pins and logic to display pins based on `type` graphQL query

#skip_new_tests

### CODE REVIEW

File: src/lib/Components/LocationMap/index.tsx [hunk 1]
```diff
@@ -9,35 +9,87 @@ import styled from "styled-components/native"
...
   render() {
-    const { location, partnerName } = this.props
+    const { location, partnerType, partnerName } = this.props
     const { lat, lng } = location.coordinates
+    const { address_2, address } = location
+    let markerAsset
+    const galleryIcon = require("../../../../images/pingalleryon.png")
+    const museumIcon = require("../../../../images/pinmuseumon.png")
+    const fairIcon = require("../../../../images/pinfairon.png")
+    if (partnerType === "Museum") {
```
[the remark this answers is not in the document]
Comment (l2succes) [reply src/lib/Components/LocationMap/index.tsx]:
```
switch (partnerType) {
 case "Fair": return fairIcon
 case "Museum": return museumIcon
 case "Gallery" return galleryIcon
}
```
 and it makes sense to break out into a separate method

File: src/lib/Components/LocationMap/index.tsx [hunk 2]
```diff
@@ -9,35 +9,87 @@ import styled from "styled-components/native"
...
+        coordinates: [lng, lat],
+      },
+      id: "mapbox-marker",
+    }
 
     return (
       <Flex>
+        <Box>
+          {partnerName && (
+            <Sans size="3" color={color("black100")} textAlign={"center"} weight="medium">
```
Comment (l2succes) [inline src/lib/Components/LocationMap/index.tsx]: `color="black100"` textAlign="center" ...`

File: src/lib/Components/LocationMap/index.tsx [hunk 3]
```diff
@@ -9,35 +9,87 @@ import styled from "styled-components/native"
 Mapbox.setAccessToken("pk.eyJ1IjoiYXJ0c3lpdCIsImEiOiJjam51dTJibTEwNHNpM3BxamV4dDRibzJhIn0.bhFMJt1mqxNDrE1xSAtxSw")
 
 const Map = styled(Mapbox.MapView)`
-  flex: 1;
-  height: 200;
+  height: 90;
```
[the remark this answers is not in the document]
Comment (kierangillen) [reply src/lib/Components/LocationMap/index.tsx]: yup!

File: src/lib/Components/LocationMap/index.tsx [hunk 4]
```diff
@@ -9,35 +9,87 @@ import styled from "styled-components/native"
...
           logoEnabled={false}
           scrollEnabled={false}
-          // zoomEnabled={false}
-        />
-        <Sans size="2" color={color("black100")} weight="medium">
-          {partnerName}
-        </Sans>
-        <Sans size="1">{location.address}</Sans>
-        <Sans size="1">{location.address_2}</Sans>
+          // zoomEnabled={true}
```
Comment (l2succes) [inline src/lib/Components/LocationMap/index.tsx]: remove commented line

File: src/lib/Scenes/Show/Components/ShowHeader/ShowHeader.tsx [hunk 5]
```diff
@@ -33,7 +38,7 @@ export class ShowHeader extends React.Component<Props> {
         <Serif size="3" textAlign="center">
           {exhibition_period}
         </Serif>
-        <Box px={2}>
```
Comment (l2succes) [inline src/lib/Scenes/Show/Components/ShowHeader/ShowHeader.tsx]: `<Box px={2} mb={2}>` would accomplish the same thing `mb = marginBottom`

Comment (kierangillen) [reply src/lib/Scenes/Show/Components/ShowHeader/ShowHeader.tsx]: Ah ok thanks!

**Comment by kierangillen** [comment]
screen shot 2018-11-20 at 4 14 42 pm

**Comment by kierangillen** [comment]
screen shot 2018-11-20 at 4 40 08 pm

File: src/__generated__/ArtistForSaleArtworksGridQuery.graphql.ts [hunk 6]
```diff
@@ -418,7 +412,13 @@ return {
...
-                          v2,
+                          {
+                            "kind": "ScalarField",
+                            "alias": null,
+                            "name": "is_biddable",
+                            "args": null,
+                            "storageKey": null
+                          },
                           {
                             "kind": "ScalarField",
```
Comment (orta) [inline src/__generated__/ArtistForSaleArtworksGridQuery.graphql.ts]: o.0

Wonder what's going on here /cc @alloy

Comment (alloy) [reply src/__generated__/ArtistForSaleArtworksGridQuery.graphql.ts]: What should I be looking at?

Comment (orta) [reply src/__generated__/ArtistForSaleArtworksGridQuery.graphql.ts]: this PR is full of these diffs doing nothing, which feels like the JSON is non-deterministic

Comment (alloy) [reply src/__generated__/ArtistForSaleArtworksGridQuery.graphql.ts]: It absolutely _should_ be deterministic. It feels like maybe field selections may have changed order at some point or something (and maybe not checked in)?

**Pull request merged**